nextflow.config

```diff
@@ -19,6 +19,7 @@ params {
 
     // Assembly options
     hifiasm_hic_on             = false
+    organelles_on              = true
```

Comment:
We are recommended to keep the default as `false` and then turn on via command line flags or the config.

Reply:
ok